lib/profile_node.stp

```diff
@@ -443,3 +443,17 @@ probe timer.profile {
         printf("\n");
 	}
 }
+
+probe timer.profile {
+    /*
+     * Since we run completely asynchronously with respect to
+     * V8's activities, we can actually catch the heap in temporarily
+     * inconsistent states, causing bad copyins and such.  Try-catch
+     * so we drop a data point in that case rather than halting.
+     */
+    try {
+        take_node_backtrace()
+    } catch {
+        next
```

Comment:
Is a data point in this context one stack trace? Is there a danger where we print half of a stack trace when take_node_backtrace throws an exception in the middle of traversing the stack?